build.sc

```diff
@@ -416,6 +422,38 @@ trait ScalaParse extends SbtModule with ScalaCliPublishModule with settings.Scal
   def scalaVersion = Scala.defaultInternal
 }
 
+trait Scala3Runtime extends SbtModule with ScalaCliPublishModule with settings.ScalaCliCompile {
```

Comment:
NIT Why do we need the `setting.` prefix here? We could just import `ScalaCliCompile` above.